### ISSUE TITLE
Over-fetching problem list

### ISSUE THREAD

**Issue by pzet123** [pzet123/WWCS-LeetCodeBot#1]
Getting the list of problems is done by fetching a set amount of problems of each category. In reality, only the problem "slugs" are needed so that when a specific problem is selected more detail can be fetched using the problem slug. Could programmatically generate a list of all problem slugs then hardcode however this comes with obvious maintainability concerns.

**Comment by pzet123** [comment]
Relevant code:
```
const easyProblems = await leetcode.problems({limit: 50, filters: {difficulty: "EASY"}});
const mediumProblems = await leetcode.problems({limit: 50, filters: {difficulty: "MEDIUM"}});
const hardProblems = await leetcode.problems({limit: 25, filters: {difficulty: "HARD"}});
const problems = easyProblems.questions.concat(mediumProblems.questions, hardProblems.questions);
```

**Comment by alfiejfs** [comment]
I think this would be fixed for the most part by #2, unless we add some other functionality to get random problems (which we may want to do in the future).

**Comment by pzet123** [comment]
Closed by #2 for now
